apps/arena/lib/arena/game_updater.ex

```diff
@@ -764,4 +765,20 @@ defmodule Arena.GameUpdater do
       end
     end)
   end
+
+  defp execute_on_explode_mechanics(game_state, projectiles) do
```

Comment:
Not for this PR, but we should probably start thinking on a `projectile.ex` module

Reply:
100% agreed, an effect module could be handy too